src/ocrd/resource_manager.py

```diff
@@ -62,6 +62,12 @@ def __init__(self, userdir=None, xdg_config_home=None, xdg_data_home=None, skip_
                 self.save_user_list()
             self.load_resource_list(self.user_list)
 
+    def __repr__(self):
+        return f"user_list={str(self.user_list)} " + \
+               f"exists={self.user_list.exists()} " + \
+               f"database: {len(self.database)} executables " + \
+               f"{sum(map(len, self.database.values()))} resources"
```

Comment:
This is a neat solution that may be used as a response when checking the resource manager's current state. I assume we no longer want to get rid of the database, or?